shared/utils/urls.ts

```diff
@@ -67,7 +67,11 @@ export function isUrl(text: string) {
  * @returns True if the url is external, false otherwise.
  */
 export function isExternalUrl(url: string) {
-  return !isInternalUrl(url);
+  if (url) {
+    return !isInternalUrl(url);
+  }
+
+  return false;
```

Comment:
I've added this to ensure that the url is not empty, since `internalUrl` method is returning false when it's empty.